.azure/pipelines/ci.yml

```diff
@@ -297,6 +297,12 @@ jobs:
     beforeBuild:
     - powershell: "& ./src/Servers/IIS/tools/UpdateIISExpressCertificate.ps1; & ./src/Servers/IIS/tools/update_schema.ps1"
       displayName: Setup IISExpress test certificates and schema
+    - powershell: "& ./.azure/pipelines/tools/SetupTestEnvironment.ps1 Setup signalrclienttests.exe"
+      displayName: Start AppVerifier
+    afterBuild:
+      - powershell: "& ./.azure/pipelines/tools/SetupTestEnvironment.ps1 Shutdown signalrclienttests.exe"
+        displayName: Stop AppVerifier
+        condition: always()
```

Comment:
nit: for consistency, reduce the indentation here and within the added `signalr-daily-tests.yml` steps